_posts/jared_baker/2017-10-20-Day-1-Collaboratory.md

```diff
@@ -0,0 +1,276 @@
+
+You can create and manage your keys from the Access & Security tab.
+
+<figure>
+    <img id="key-list" src="{{site.urlimg}}jared_baker/day1collab/2-key-list.png" data-featherlight="#key-list" />
+</figure>
+
+You can also view the detail of your keys and see the public key pair which is used on the instances you create and stored in the Collaboratory.
+
+<figure>
+    <img id="key-detail" src="{{site.urlimg}}jared_baker/day1collab/2-key-detail.png" data-featherlight="#key-detail" />
+</figure>
+
+<a id="3"></a>
+## Step 3: Create an instance
+Instances are Virtual Machines, aka VM's. Let's begin creating one by clicking 'Launch Instance' under the 'Instances' tab. This will launch a wizard where we will need to provide some additional information before it can be created.
+
+Provide a useful name to identify the instance by. Leave the count at 1. If we wanted more instances like the one we are creating we could put the number desired here and your instances would be named dynamically. Example, if you instance name was 'my-first-instance' and the count was 3, you would have 3 instances with names 'my-first-instance-1, my-first-instance-2, my-first-instance-3'.
+
+<figure>
+    <img id="instance-name" src="{{site.urlimg}}jared_baker/day1collab/3-instance-name.png" data-featherlight="#instance-name" />
+</figure>
+
+The Source tab is where you will choose the Operating System that your instance will run. Ensure that your boot source is **'Image'** and not 'Volume' and that 'Create New Volume' is set to **'No'** This will make sure that your instance is provisioned correctly and will use the local disk of the underlying compute host. An instance backed by a Volume will have lower disk I/O performance and will incur extra charges.
+
+<figure>
+    <img id="instance-source" src="{{site.urlimg}}jared_baker/day1collab/3-instance-source.png" data-featherlight="#instance-source"/>
+</figure>
+
+Instances in the Collaboratory come in different *flavours*. Flavours are what defines the specifications (# of CPU, RAM, and local disk space) of the instance. Choose a flavour to meet your computing needs.
+
+<figure>
+    <img id="instance-flavor" src="{{site.urlimg}}jared_baker/day1collab/3-instance-flavor.png" data-featherlight="#instance-flavor" />
+</figure>
+
+Select a Network for your instance to communicate on. New projects already have a network created for them, but you can create more if needed.
+
+<figure>
+    <img id="instance-networks" src="{{site.urlimg}}jared_baker/day1collab/3-instance-networks.png" data-featherlight="#instance-networks"/>
+</figure>
+
+Security groups the firewall for your instance. They define what can and cant go to/from your Instance over the network. Choose the default security group for now as we will modify this later on.
```

Comment:
"Security groups the firewall for your instance" -> what does it mean?
"go to/from your Instance over"  -> "Instance" should be in lower case